operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1927,32 +1927,33 @@ metaMonitoring:
       # configuration to push metrics to this Prometheus-compatible remote. Optional.
       # Note that you need to configure serviceMonitor in order to have some metrics available.
       #
-      # You can also send the collected metamonitoring metrics to the installation of Mimir or GEM.
-      # The configuration varies slightly for GEM and Mimir.
+      # If you leave the metamonitoring.grafanaAgent.metrics.remote.url field empty,
+      # then the chart automatically fills in the address of the GEM gateway Service
+      # or the Mimir NGINX Service.
       #
-      # If you have deployed Mimir, then the Helm chart values should look like the following example.
-      # Replace `HELM_RELEASE_NAME` with the name of your Helm release:
+      # If you have deployed Mimir, and metamonitoring.grafanaAgent.metrics.remote.url is not set,
+      # then the metamonitoring metrics are be sent to the Mimir cluster.
+      # You can query these metrics using X-Scope-OrgID: metamonitoring
       #
-      # remote:
-      #   url: 'http://<HELM_RELEASE_NAME>-mimir-nginx.mimir.svc/api/v1/push'
-      #   headers:
-      #     X-Scope-OrgID: metamonitoring
+      # If you have deployed GEM, then there are two cases:
+      # * If are using the 'trust' authentication type (mimir.structuredConfig.auth.type: trust),
+      #   then the same instructions apply as for Mimir.
       #
-      # If you have deployed GEM, then the URL will point to the GEM gateway instead of the nginx Deployment.
-      # If you are using the GEM authentication model, then you also need to provide a Secret with the
-      # authentication token for a tenant. Refer to https://grafana.com/docs/mimir/latest/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs/#collect-metrics-and-logs-via-the-helm-chart
-      # for setting up the Secret.
-      # Assuming you are using the GEM authentication model, the Helm chart values should look like the following example.
-      # Replace `HELM_RELEASE_NAME` with the name of your Helm release:
+      # * If you are using the enterprise authentication type (mimir.structuredConfig.auth.type: enterprise, which is also the default),
```

Comment:
```suggestion
      # * If you are using the enterprise authentication type (mimir.structuredConfig.auth.type: enterprise, which is also the default when enterprise.enabled=true),
```

Reply:
aren't we in the context of GEM anyways?